README.md

```diff
@@ -32,7 +32,7 @@
 
 <!-- PROJECT LOGO -->
 <div align="center">
-  <img src="./logo.png" alt="Logo" width="200" height="200">
+  <img src="./docs/static/img/logo.png" alt="Logo" width="200" height="200">
```

Comment:
I think we need to fix some other links in here if the md guides are moving

Reply:
I think most of the content of this file should probably be removed and kept only in our docs now.